eng/Version.Details.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="9.0.0-alpha.1.23475.1">
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="9.0.0-alpha.1.23502.1">
```

Comment:
This seems concerning--shouldn't we be on the 8.0.0 sourcebuild intermediates, since our `main` branch will ship in 8.0.200 next?

Reply:
Your dotnet/source-build-reference-packages subscription seems to be from ".NET Eng - Latest", which ultimately goes into sdk main, which is 9.0 at this point. I think it should be changed to ".NET 8 Eng".

On the note of darc subscriptions, you might also want to update most of your other ones as well. Arcade is also pulling from latest, and roslyn and nuget are pulling from VS 17.8 instead of VS 17.9.